backend/src/main/java/gov/cdc/usds/simplereport/service/model/DeviceSpecimenTypeHolder.java

```diff
@@ -0,0 +1,26 @@
+package gov.cdc.usds.simplereport.service.model;
+
+import java.util.Collections;
+import java.util.List;
+
+import gov.cdc.usds.simplereport.db.model.DeviceSpecimenType;
+
+public class DeviceSpecimenTypeHolder {
+
+    private DeviceSpecimenType _defaultDeviceType;
+    private List<DeviceSpecimenType> _allDeviceTypes;
+
+    public DeviceSpecimenTypeHolder(DeviceSpecimenType defaultType, List<DeviceSpecimenType> configuredTypes) {
+        super();
+        this._defaultDeviceType = defaultType;
+        this._allDeviceTypes = Collections.unmodifiableList(configuredTypes);
```

Comment:
i suppose given the nature of this PR, this is not merely a nitpick: should be `_allDeviceSpecimenTypes`

Reply:
That is precisely correct. 🤣